docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -198,6 +198,13 @@ There are different types of Bareos binaries:
 
 The software in both types of repositories is based on the same source code freely available in https://github.com/bareos/bareos/. There are no hidden nor open core components.
 
+.. note::
+
+   About (``<next-version-number>~pre``) for example if we have released 22.0.1 and some code has been backported in the meantime to the current (22) version,
+   newer packages with a 22.0.2~pre version will get published. It means latest stable current 22.0.1 release + newer backported feature that will appear in 22.0.2.
```

Comment:
```suggestion
   About ``<next-version-number>~pre`` version numbers:
   
   Assume the current major release is ``22`` and the latest stable release is ``22.0.1``.
   As soon as some code gets backported to Bareos 22,
   packages with ``22.0.2~pre`` version will get published.
   It means the latest stable current ``22.0.1`` release + newer backported feature that will appear in ``22.0.2``.
```

Does this note not only belong to the https://download.bareos.org/current/ part?

Reply:
yes only d.b.o/current is concerned, I've tried to bring the note just after but it make the split between the two type of repo less evident.